MonumentAddons.cs

```diff
@@ -5320,6 +5328,18 @@ private void DisableVehicleDecay(BaseEntity vehicle)
                     return;
                 }
             }
+
+            private void SetHackableCrateDecay(HackableLockedCrate hackableCrate)
+            {
+                if (hackableCrate == null)
+                    return;
```

Comment:
This code can be removed since this condition shouldn't be possible, as invokes will not be called for destroyed objects.

Reply:
My thinking was that method doesn't know from where it was called, so best practice would be to nullcheck just to be sure, but I agree that there is close to zero chance that it would be called from elsewhere.